app/views/docs/databases.phtml

```diff
@@ -21,8 +21,7 @@
 
 <h2><a href="/docs/databases#collection" id="collection">Create Your Collections</a></h2>
 <p>
-    Appwrite uses collections as containers of documents.
-    The terms collections and documents are used because the Appwrite JSON REST API resembles the API of a traditional NoSQL database. That said, internally, Appwrite can support both SQL and NoSQL database adapters like MariaDB, MySQL, or MongoDB. When working with an SQL adapter, Appwrite will treat your collections as tables and documents as rows on native SQL tables.
+    Appwrite uses collections as containers of documents. The terms collections and documents are used because the Appwrite JSON REST API resembles the API of a traditional NoSQL database. That said, internally, Appwrite will support both SQL and NoSQL database adapters like MariaDB, MySQL, or MongoDB. When working with an SQL adapter, Appwrite will treat your collections as tables and documents as rows on native SQL tables.
```

Comment:
I'm not sure why we had 2 lines of text before...i basically just changed:

> Appwrite can support both...

to 

> Appwrite will support both...